homeassistant/components/airzone/select.py

```diff
@@ -32,27 +32,26 @@ class AirzoneSelectDescriptionMixin:
     """Define an entity description mixin for select entities."""
 
     api_param: str
-    options_dict: dict[str, int]
 
 
 @dataclass
 class AirzoneSelectDescription(SelectEntityDescription, AirzoneSelectDescriptionMixin):
     """Class to describe an Airzone select entity."""
 
 
-GRILLE_ANGLE_DICT: Final[dict[str, int]] = {
-    "90º": GrilleAngle.DEG_90,
-    "50º": GrilleAngle.DEG_50,
-    "45º": GrilleAngle.DEG_45,
-    "40º": GrilleAngle.DEG_40,
-}
+GRILLE_ANGLE_OPTIONS: Final[list[str]] = [
+    str(GrilleAngle.DEG_90.value),
```

Comment:
I don't think we should use the library enum to define the integration options since that enum is not defined in the integration but in the library. It could change in a library update without an explicit change in the integration. The integration should have strict control of the options.

Reply:
The integration library defines the values as they are documented in the API documentation.
Therefore, those values won't change at all, since the Airzone API is the same for every machine and they always keep it compatible with older devices.
However, I can change it back to a dict and undo the changes requested by @gjohansson-ST

Reply:
Yes, please change it back.

Reply:
Done:
- fea174b4567f18c64a6b354780c81713de802ce5
- 36c0b15900bc2c3a1b2d3f6a0957ee27261ee55b
- 65ff994393f84e11bedfdb9fb0aa3f146257e72b